pkg/controllers/dynakube/extension/eec/reconciler_test.go

```diff
@@ -196,7 +196,7 @@ func TestEnvironmentVariables(t *testing.T) {
 		assert.Equal(t, corev1.EnvVar{Name: envExtensionsModuleExecPathName, Value: envExtensionsModuleExecPath}, statefulSet.Spec.Template.Spec.Containers[0].Env[4])
 		assert.Equal(t, corev1.EnvVar{Name: envDsInstallDirName, Value: envDsInstallDir}, statefulSet.Spec.Template.Spec.Containers[0].Env[5])
 		assert.Equal(t, corev1.EnvVar{Name: envK8sClusterId, Value: dk.Status.KubeSystemUUID}, statefulSet.Spec.Template.Spec.Containers[0].Env[6])
-		assert.Equal(t, corev1.EnvVar{Name: envK8sExtServiceUrl, Value: serviceAccountName}, statefulSet.Spec.Template.Spec.Containers[0].Env[7])
+		assert.Equal(t, corev1.EnvVar{Name: envK8sExtServiceUrl, Value: "dynakube-extensions-controller.dynatrace"}, statefulSet.Spec.Template.Spec.Containers[0].Env[7])
```

Comment:
please replace with constant as other asserts.

Reply:
done for consistency reasons in 6c108872af9035d9124d0ac5c6bc4565b5ae7c74

I actually think calculating an expected value in a test to be bad practice.